### PR TITLE
Update image related dependencies

### PR DESCRIPTION
**Pull request by Eijebong**
This change is [Reviewable](https://reviewable.io/reviews/servo/servo/22209)

### CODE REVIEW

**Comment by jdm** [comment]
screen shot 2018-11-16 at 10 44 21 am

There's a line at the top of each of the highlighted boxes that is #aaaaaa instead of #a9a9a9a9 now.

**Comment by jdm** [comment]
Oddly, the reference and non-reference use the exact same image, just laid out in different ways, so I am surprised that there's a rendering difference when updating the image loading deps.

**Comment by jdm** [comment]
https://searchfox.org/mozilla-central/source/testing/web-platform/tests/css/css-backgrounds/background-repeat/background-repeat-round-roundup.xht vs https://searchfox.org/mozilla-central/source/testing/web-platform/tests/css/css-backgrounds/background-repeat/reference/background-repeat-round-roundup.xht

File: servo-tidy.toml [hunk 1]
```diff
@@ -32,15 +33,12 @@ rand = [
 [ignore]
 # Ignored packages with duplicated versions
 packages = [
-  "core-graphics",
   "crossbeam-deque",
   "crossbeam-epoch",
   "crossbeam-utils",
   "gl_generator",
```
Comment (atouchet) [inline servo-tidy.toml]: A few duplicate crates were introduced in #22202. Those can probably be removed now.

Comment (Eijebong) [reply servo-tidy.toml]: Yup, forgot about them :p Will remove them once I'm done with the other PR I'm rebasing

**Comment by Eijebong** [comment]
Apparently I broke skia on windows... Trying to fix it

@bors-servo try

**Comment by Eijebong** [comment]
`{"status": "PASS", "group": "default", "message": null, "stack": null, "subtest": null, "test": "/css/css-backgrounds/background-repeat/background-repeat-round-roundup.xht", "line": 2037, "action": "test_result", "expected": "FAIL"}`

This is an intermittent then ;_;

**Comment by Eijebong** [comment]
https://github.com/servo/skia/pull/169

**Review by jdm — APPROVED** [review]

**Comment by Eijebong** [comment]
Should be good to go now

**Pull request merged**